catalogue/webapp/pages/work.js

```diff
@@ -222,7 +237,18 @@ WorkPage.getInitialProps = async (
 
   const { id, query, page } = ctx.query;
   const workOrError = await getWork({ id });
+  const iiifPresentationLocation = getIIIFPresentationLocation(workOrError);
 
+  let iiifPresentationDownloadOptions = [];
+  if (iiifPresentationLocation) {
+    try {
+      const iiifManifest = await fetch(iiifPresentationLocation.url);
+      const manifestData = await iiifManifest.json();
+      iiifPresentationDownloadOptions = getDownloadOptionsFromManifest(
+        manifestData
+      );
+    } catch (e) {}
```

Comment:
A little worried about relying on on the DCLS stuff here.
We could do it client side, and only when someone clicks to the button to save on bandwidth.

I wonder if there is a very lo-fi version where we just render the links on the screen at `/works/{id}/downloads`?

Is there a reason to do parsing of the options in the getInitialProps over doing it in the component?